pmda/parallel.py

```diff
@@ -313,26 +315,31 @@ def run(self,
         start, stop, step = self._trajectory.check_slice_indices(
             start, stop, step)
         n_frames = len(range(start, stop, step))
-        bsize = int(np.ceil(n_frames / float(n_blocks)))
+        if n_frames == 0:
+            warnings.warn("run() analyses no frames: check start/stop/step")
```

Comment:
Do we need to add a warning when n_frames < n_blocks ?

Reply:
We probably should. 

This is the only case when we currently produce slices with 0 frames. It still works, I think...

Reply:
added warning + tes